server.js

```diff
@@ -985,6 +984,46 @@ routes['join_room'] = function (user, data) {
     room.sync(user);
 }
 
+routes['search_youtube'] = function (user, data) {
+	if(user.room === null) return;
+	var room = user.room;
+
+	var query = encodeURIComponent(data.query);
+	if (typeof(data.query === 'string') && query.length > 0 && query.length < 101) {
```

Comment:
yeah, this seems like a bug, should be typeof(data.query) === 'string'. Also what other type would this be?